project/Dependencies.scala

```diff
@@ -1,13 +1,13 @@
 import sbt._
 
 object Dependencies {
-  val akkaV = "2.5.19"
-  val akkaHttpV = "10.1.7"
+  val akkaV = "2.5.31"
+  val akkaHttpV = "10.1.12"
   val slickV = "3.3.2"
 
   val googleV = "1.22.0"
   val olderGoogleV = "1.20.0"   // TODO why do we have two google versions?  GAWB-2149
-  val workbenchGoogle2V = "0.6-4df42ca"
+  val workbenchGoogle2V = "0.10-ae0b940"
```

Comment:
we want this upgrade to take advantage of the newer BigQuery client from Google.